cmd/endorse.go

```diff
@@ -38,9 +41,12 @@ import (
 // endorseCommand stores the common flag values for the endorse subcommand that isn't directly
 // represented in endorse.Context.
 type endorseCommand struct {
-	AddSnp   bool
-	AddTdx   bool
-	UefiPath string
+	Storage             storagei.Client
+	AddSnp              bool
+	AddTdx              bool
+	UefiPath            string
+	SvsmPath            string
+	SvsmMeasurementPath string
```

Comment:
```suggestion
	IGVMPath            string
	IGVMSNPMeasurementPath string
```

Reply:
I'm not sure IGVM is the correct abstraction here. In theory, we could use the IGVM for for non-SVSM firmwares as well (instead of .fd). If we did that, we'd still want to distribute the OVMF separately from the SVSM+OVMF since we'd support both products. So I think it's better to keep calling it SVSM rather than IGVM.